### PR TITLE
change_password_3.0

### PR DESCRIPTION
**Pull request by yinxuan**

### CODE REVIEW

File: server/server/www/web/templates/change_password.html [hunk 1]
```diff
@@ -3,47 +3,112 @@
 <head>
     <meta charset="UTF-8">
     <title>{{ title }}</title>
+	<script src="/static/node_modules/jquery/dist/jquery.min.js"></script>
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:6]: Are you sure your jquery is really imported?

File: server/server/www/web/templates/change_password.html [hunk 2]
```diff
@@ -3,47 +3,112 @@
...
+	<link rel="stylesheet" href="https://cdn.bootcss.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:7]: what's `integrity`, `crossorigin`? why need them? what if not?

File: server/server/www/web/templates/change_password.html [hunk 3]
```diff
@@ -3,47 +3,112 @@
...
-<link rel="stylesheet" href="https://cdn.bootcss.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+	<style type="text/css">
+		body {
+			background-color: #FAF9F3;
+			background-repeat:repeat;
+		}
 
-<!-- 最新的 Bootstrap 核心 JavaScript 文件 -->
-<script src="https://cdn.bootcss.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+		.PassWord_Change {
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:17]: css class name shouldn't be `PassWord_Change`, it's better to use `password_change`. You can read the source code of `https://google.com` or `https://baidu.com` page. Learn from good code.

File: server/server/www/web/templates/change_password.html [hunk 4]
```diff
@@ -3,47 +3,112 @@
...
+
+
+		}
+
+		@keyframes icon-bounce {
+		  0%, 100% {
+		    -moz-transform: rotate(0deg);
+		    -ms-transform: rotate(0deg);
+		    -webkit-transform: rotate(0deg);
+		    transform: rotate(0deg);
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:34]: what's the difference between these four `-transform`? just curious.

File: server/server/www/web/templates/change_password.html [hunk 5]
```diff
@@ -3,47 +3,112 @@
...
+			<div style="margin-top: 50px;">
+				<input type="text" placeholder="忘记密码" style="border: none;background-color: #FAF9F3;outline: none;">
+			</div>
+			<div style="background: linear-gradient(to right, white , #f5a623 , white);height: 2px;margin-bottom: 20px;"></div>
+			<div style="margin-top: 50px;">
+				<input type="text" placeholder="确认密码" style="border: none;background-color: #FAF9F3;outline: none;">
+			</div>	
+			<div style="background: linear-gradient(to right, white , #f5a623 , white);height: 2px;margin-bottom: 20px;"></div>
+			<div>
+				<button id="btn_ChangePassWord" class="btn btn-primary" type="submit" style="margin-top: 20px;margin-bottom: 20px;background-color: red;border-color: red">Seed password reset email</button>
```
Comment (wingjay) [inline server/server/www/web/templates/change_password.html:110]: Not `Seed`, `Send`

**Review by wingjay — CHANGES_REQUESTED** [review]

File: server/server/www/web/templates/change_password.html (hunk 1 above)
Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:6]: <script type="text/javascript" src="jquery.js"></script>

File: server/server/www/web/templates/change_password.html (hunk 2 above)
Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:7]: 不需要也可以

File: server/server/www/web/templates/change_password.html (hunk 3 above)
Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:17]: okay

File: server/server/www/web/templates/change_password.html (hunk 4 above)
Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:34]: 适用于不同内核的浏览器，不然某些类似IE6之流的实现不了动画效果

File: server/server/www/web/templates/change_password.html (hunk 5 above)
Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:110]: okay

File: server/server/www/web/templates/change_password.html (hunk 4 above)
Comment (wingjay) [reply server/server/www/web/templates/change_password.html:34]: 现在还要适配IE6？？

File: server/server/www/web/templates/change_password.html (hunk 2 above)
Comment (wingjay) [reply server/server/www/web/templates/change_password.html:7]: 干嘛用的？

File: server/server/www/web/templates/change_password.html (hunk 1 above)
Comment (wingjay) [reply server/server/www/web/templates/change_password.html:6]: 这样就可以了？jquery.js文件呢？

Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:6]: <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
这个是最新的

File: server/server/www/web/templates/change_password.html (hunk 4 above)
Comment (yinxuan) [reply server/server/www/web/templates/change_password.html:34]: 谁知道上这个网站的人用什么浏览器，有备无患

File: server/server/www/web/templates/change_password.html (hunk 2 above)
Comment (wingjay) [reply server/server/www/web/templates/change_password.html:7]: 求解答

**Pull request merged**